src/main/java/it/gov/pagopa/common/web/exception/ServiceException.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.common.web.exception;
+
+
+import it.gov.pagopa.reward.dto.synchronous.SynchronousTransactionResponseDTO;
+import lombok.Getter;
+
+@Getter
+public class ServiceException extends RuntimeException {
+    private final String code;
+    private final boolean printStackTrace;
+    private final transient SynchronousTransactionResponseDTO response;
```

Comment:
let's remove the dependency from the "reward" class
```suggestion
public class ServiceException <T> extends RuntimeException {
    private final String code;
    private final boolean printStackTrace;
    private final transient T response;
```